src/openms/source/ANALYSIS/OPENSWATH/TargetedSpectraExtractor.cpp

```diff
@@ -373,6 +372,10 @@ namespace OpenMS
     const bool compute_features
   ) const
   {
+    if (compute_features && scored_spectra.size() != features.size())
+    {
+      throw Exception::InvalidSize(__FILE__, __LINE__, OPENMS_PRETTY_FUNCTION);
```

Comment:
can you use `TEST_EXCEPTION` to test this code path?

Reply:
Thanks for merging. I'll do this in some other PR, noted :+1: